docs/source/federation-versions.mdx

```diff
@@ -21,6 +21,103 @@ This article describes notable changes and additions introduced in each minor ve
 
 - **If you maintain a [subgraph-compatible library](./building-supergraphs/supported-subgraphs/),** consult this article to stay current with recently added directives. All of these directive definitions are also listed in the [subgraph specification](./subgraph-spec/#subgraph-schema-additions).
 
+## v2.6
+
+<hr/>
+
+<CodeColumns cols="3">
+
+<div>
+
+First release
+
+**November 2023**
+
+</div>
+
+<div>
+
+Available in GraphOS?
+
+**Yes**
+
+</div>
+
+<div>
+
+Minimum router version
+
+**`1.35.0`**
+
+</div>
```

Comment:
@Geal - what should this `Minimum router version` be for `@policy` support with Federation 2.6.0?

Reply:
Compatible router wasn't released yet (and this feature is not yet available in GraphOS). We can update this once router is released and it is available in studio.

Reply:
@dariuszkuc thanks, changing for now to unavailable in GraphOS and min router version of 1.34.1

Reply:
1.34.1 is not compatible, 1.35.0 (unreleased) might be

Reply:
1.34.1 is not compatible, 1.35.0 might be but is still unreleased. IMHO we should just put TBD and fix it once we got router release

Reply:
@dariuszkuc as suggested, setting to TBD for now